src/main/kotlin/com/makarytskyi/rentcar/dto/order/UpdateOrderRequest.kt

```diff
@@ -0,0 +1,21 @@
+package com.makarytskyi.rentcar.dto.order
+
+import com.makarytskyi.rentcar.model.Order
+import jakarta.validation.constraints.NotNull
+import java.util.Date
+
+data class UpdateOrderRequest(
+    val from: Date?,
+    val to: Date?,
+) {
+
+    companion object {
+        fun toEntity(orderRequest: UpdateOrderRequest) = Order(
+            null,
+            null,
+            null,
```

Comment:
if you're setting null or some literal - please use named arguments. because to understand which 3 parameters you're ignoring reader should navigate to Order class